test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -12,17 +12,7 @@
 import org.junit.Before;
 import org.junit.Test;
 
-import seedu.addressbook.commands.AddCommand;
-import seedu.addressbook.commands.ClearCommand;
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```

Comment:
According to the [Google Style Guide which we are using for this course](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports), wildcard imports is not used.

Reply:
Ah yes I overlooked this, will be more careful next time! 😅

Reply:
Ah yes I overlooked this, will be more careful next time! 😅